nexus/db-queries/src/db/datastore/volume.rs

```diff
@@ -1681,13 +1992,123 @@ pub fn volume_is_read_only(
     }
 }
 
+/// Replace a Region in a VolumeConstructionRequest
+///
```

Comment:
This only does something if the VCR is Volume or Region.
Do we want the other types to return error instead of doing nothing?

Reply:
I'm ok having this do nothing right now. We won't see those variants in production any more.

Reply:
I vote for `panic!`
But, I'll leave this to you.